domain/src/main/java/org/oppia/android/domain/exploration/ExplorationProgressController.kt

```diff
@@ -255,9 +282,7 @@ class ExplorationProgressController @Inject constructor(
     }
   }
 
-  fun submitSolutionIsRevealed(
-    state: State
-  ): LiveData<AsyncResult<Solution>> {
+  fun submitSolutionIsRevealed(): LiveData<AsyncResult<Solution>> {
```

Comment:
Ditto for this method (please add a KDoc).

Reply:
done